src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const persons = document.querySelectorAll('ul li');
```

Comment:
You could create only one variable 'list' and then use it like `[...list.children]` to get the array of employees instead of creating the variable 'persons'